### PR TITLE
Fix trim headers values

### PR DESCRIPTION
**Pull request by Nicholas122**
Fix Type error: trim() expects parameter 1 to be string, integer given" at /vendor/nyholm/psr7/src/MessageTrait.php line 216

### CODE REVIEW

File: src/MessageTrait.php [hunk 1]
```diff
@@ -177,7 +177,7 @@ private function setHeaders(array $headers)
     private function trimHeaderValues(array $values)
     {
         return array_map(function ($value) {
-            return trim($value, " \t");
+            return trim(strval($value), " \t");
```
Comment (prisis) [inline src/MessageTrait.php]: Why not (string) $value ?

**Comment by Tobion** [comment]
This error can only happen if you use strict types which is not enabled in this repo.

> /vendor/nyholm/psr7/src/MessageTrait.php line 216

You are using a fork apperently.

**Comment by Nyholm** [comment]
I see that you are using `nyholm/psr7`

**Comment by Tobion** [comment]
Fixed in #272

**Pull request closed without merging**